addons/account_peppol/models/account_move.py

```diff
@@ -50,3 +50,7 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = False
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    def _default_enabled_peppol(self):
+        self.ensure_one()
+        return self.peppol_move_state == 'ready'
```

Comment:
The need to create this function is arguable, I can directly use the condition in the compute.
But since we have that in many modules, I thought it might be a good idea to keep the practice.

Reply:
What about something a bit more explicit: `_is_peppol_enabled_by_default` ?
Also, can you add a bit of docstring saying it's mainly use when using the send and print?